pkg/doc/verifiable/presentation.go

```diff
@@ -178,49 +179,58 @@ func (vp *Presentation) JWTClaims(audience []string, minimizeVP bool) *JWTPresCl
 	return newJWTPresClaims(vp, audience, minimizeVP)
 }
 
-// Credentials provides Verifiable Credentials enclosed into Presentation in raw byte array format.
-func (vp *Presentation) Credentials() ([]PresentationCredential, error) {
-	marshalSingleCredFn := func(cred interface{}) (PresentationCredential, error) {
-		credBytes, err := json.Marshal(cred)
-		if err != nil {
-			return nil, fmt.Errorf("marshal credentials from presentation: %w", err)
-		}
+// Credentials returns current credentials of presentation.
+func (vp *Presentation) Credentials() []interface{} {
+	return vp.credentials
+}
 
-		return credBytes, nil
+// SetCredentials defines credentials of presentation.
+// The credential could be string/byte (probably serialized JWT) or Credential structure.
+func (vp *Presentation) SetCredentials(creds ...interface{}) error {
+	for i := range creds {
+		switch creds[i].(type) {
+		default:
+			return errors.New("unsupported credential format")
+		case []byte, string, *Credential, Credential:
```

Comment:
Empty case statement?

Reply:
Yes, it's done on purpose in order to separate OK case (do nothing) with default exceptional one (do not accept VC of such kind).
I will make this more precise by putting `default: ` at the bottom of `switch` (what is everyone used for) and add a comment on empty case.